### PR TITLE
Add change variable type code action in module level

### LINKED ISSUE #36682: Change variable type code action not working in module level
**Description:**
$Title

**Steps to reproduce:**
Screenshot 2022-06-23 at 09 34 13

Screenshot 2022-06-23 at 09 35 47

```
const int OIL_PRICE = 10.0;
```
```
int OIL_PRICE = 10.0
```
**Affected Versions:**
2201.1.1-RC-1

### PR DESCRIPTION
**Pull request by bimsarapathiraja**
## Purpose
Change variable code action is not available in modular level

Fixes #36682

## Check List 
- [x] Read the [Contributing Guide](https://github.com/ballerina-platform/ballerina-lang/blob/master/CONTRIBUTING.md)
- [ ] Updated Change Log
- [ ] Checked Tooling Support (#<Issue Number>)
- [ ] Added necessary tests
  - [x] Unit Tests
  - [ ] Spec Conformance Tests
  - [ ] Integration Tests
  - [ ] Ballerina By Example Tests
- [ ] Increased Test Coverage   
- [ ] Added necessary documentation  
  - [ ] API documentation 
  - [ ] Module documentation in Module.md files
  - [ ] Ballerina By Examples

### CODE REVIEW

File: language-server/modules/langserver-core/src/main/java/org/ballerinalang/langserver/codeaction/providers/changetype/ChangeVariableTypeCodeAction.java [hunk 1]
```diff
@@ -187,6 +192,14 @@ private Optional<String> getVariableName(Node matchedNode) {
                 }
                 CaptureBindingPatternNode captureBindingPatternNode = (CaptureBindingPatternNode) bindingPatternNode;
                 return Optional.of(captureBindingPatternNode.variableName().text());
+            case MODULE_VAR_DECL:
+                 ModuleVariableDeclarationNode mVariableDeclrNode = (ModuleVariableDeclarationNode) matchedNode;
+                 BindingPatternNode mBindingPatternNode = mVariableDeclrNode.typedBindingPattern().bindingPattern();
```
Comment (malinthar) [inline language-server/modules/langserver-core/src/main/java/org/ballerinalang/langserver/codeaction/providers/changetype/ChangeVariableTypeCodeAction.java:197]:
```suggestion
                 BindingPatternNode mBindingPatternNode = ((ModuleVariableDeclarationNode) matchedNode).typedBindingPattern().bindingPattern();
```

File: language-server/modules/langserver-core/src/main/java/org/ballerinalang/langserver/codeaction/providers/changetype/ChangeVariableTypeCodeAction.java [hunk 2]
```diff
@@ -187,6 +192,14 @@ private Optional<String> getVariableName(Node matchedNode) {
...
+                 if (mBindingPatternNode.kind() != SyntaxKind.CAPTURE_BINDING_PATTERN) {
+                     return Optional.empty();
+                 }
+                 CaptureBindingPatternNode mCaptureBindingPatternNode = (CaptureBindingPatternNode) mBindingPatternNode;
+                 return Optional.of(mCaptureBindingPatternNode.variableName().text());
```
Comment (malinthar) [inline language-server/modules/langserver-core/src/main/java/org/ballerinalang/langserver/codeaction/providers/changetype/ChangeVariableTypeCodeAction.java:202]:
```suggestion
                 return Optional.of(((CaptureBindingPatternNode) mBindingPatternNode).variableName().text());
```

File: language-server/modules/langserver-core/src/test/resources/codeaction/change-var-type/source/changeVarType_int_to_float.bal [hunk 3]
```diff
@@ -0,0 +1 @@
+int x = 10.0;
```
Comment (malinthar) [inline language-server/modules/langserver-core/src/test/resources/codeaction/change-var-type/source/changeVarType_int_to_float.bal:1]: New line

File: language-server/modules/langserver-core/src/test/resources/codeaction/change-var-type/config/changeVarType_int_to_float.json [hunk 4]
```diff
@@ -0,0 +1,28 @@
...
+               "line": 0,
+               "character": 3
+             }
+           },
+           "newText": "float"
+         }
+       ]
+     }
+   ]
+ }
```
Comment (malinthar) [inline language-server/modules/langserver-core/src/test/resources/codeaction/change-var-type/config/changeVarType_int_to_float.json:28]: Let's add a new line

**Review by malinthar — CHANGES_REQUESTED** [review]

**Review by gayalkuruppu — CHANGES_REQUESTED** [review]
Shall we add a test case for constant as well? The test case is mentioned in the original issue.  And we can include a `description` in the config file as well.

**Review by malinthar — DISMISSED** [review]
LGTM!

**Review by gayalkuruppu — APPROVED** [review]
LGTM

**Pull request merged**